steps/testfairy-deploy-android/1.0.0/step.yml

```diff
@@ -0,0 +1,124 @@
+title: TestFairy Deploy Android
+summary: |-
+  Deploy your awesome App to TestFairy, set Tester Groups to be notified,
+  enable email notification for the build, enable in-app notification of new
+  version, set recording duration, set if video recording is enabled and add
+  your comments and let the TestFairy do it's magic.
+description: |-
```

Comment:
On bitrise.io setp's description will be displayed, maybe it would be better to provide here the same description as in summary too, to provide little bit more info about the step.
In general both bitrise-cli and bitrise.io use step's description, so its better to provide more details in description.